app/models/member.rb

```diff
@@ -48,9 +48,13 @@ def banned_permanently?
     bans.permanent.present?
   end
 
+  def name_and_surname
+    [name, surname].compact.join ' '
+  end
```

Comment:
I found it handy to have a method without pronouns for the organisers’ interface, for brevity.